### DISCUSSION TITLE
is it possible to make visual novels work?

### DISCUSSION THREAD

**Discussion by frans4080** [skyline-emu/skyline#478] [General]
i'm guessing it shouldn't be hard to make them work since there isn't alot of 3d rendering infact there isn't any 3d rendering.

**Comment by Tanin13** [comment]
Yup i also want them to work

**Comment by PixelyIon** [comment]
Visual novels are too generic of a category and rendering does vary from engine to engine, it would be better to create issues on a per-game basis.
